### PR TITLE
bugfix/lifecycle-notifications-adjustments

### PR DESCRIPTION
**Pull request by baywet**
adjustments for the late comments in #8965

### CODE REVIEW

File: concepts/changelog.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ For a summary of the value of these API changes, as well as recent tools, compon
 
 | **Change type** | **Version**   | **Description**                          |
 | :-------------- | :------------ | :--------------------------------------- |
-| Addition | beta and v1.0 | Added [lifecycle notifications](./webhooks-lifecycle.md).|
+| Addition | v1.0 | Added [lifecycle notifications](./webhooks-lifecycle.md) for personal [chatMessage][/graph/api/resources/chatmessage?view=graph-rest-1.0], [contact][/graph/api/resources/contact?view=graph-rest-1.0], [event][/graph/api/resources/event?view=graph-rest-1.0], and [message][/graph/api/resources/message?view=graph-rest-1.0].|
```
Comment (angelgolfer-ms) [inline concepts/changelog.md:32]: @baywet , does chatMessage in beta support lifecycle notifications as well? 
If so, is beta support also new?
If beta support is new, then we should add that as a separate table row for the beta version.

Comment (angelgolfer-ms) [reply concepts/changelog.md:32]: Also, the links are not properly formatted with curly brackets, please fix.

File: concepts/webhooks-lifecycle.md [hunk 2]
```diff
@@ -27,6 +27,15 @@ An app subscribing to change notifications should listen to the `subscriptionRem
 
 To receive lifecycle notifications, you can use the existing **notificationUrl** endpoint that already receives change notifications, or you can register a separate **lifecycleNotificationUrl** to receive `subscriptionRemoved` and `missed` lifecycle notifications in a separate endpoint.
 
+Lifecycle notifications are emitted for subscriptions created on these resource types:
```
Comment (angelgolfer-ms) [inline concepts/webhooks-lifecycle.md:30]: Line 30:
Please replace "emitted" by "supported", to use a more commonly used term, per Microsoft Voice: https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=29173

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
@baywet , thank you so much for patching up with this PR!
Just three minor comments, please take a look. 
Apart from those issues, the PR looks good to be merged.

**Review by Lauragra — APPROVED** [review]

File: concepts/changelog.md (hunk 1 above)
Comment (baywet) [reply concepts/changelog.md]: @Lauragra thanks for fixing the links.
@angelgolfer-ms thanks for the feedback. lifecycle notifications has been in beta for chatMessages, events, messages and contacts for almost a year now. There should have been a changelog entry created by Peter back then. The only changes we're introducing at this point is the GA of that capability on those resources.

File: concepts/webhooks-lifecycle.md (hunk 2 above)
Comment (baywet) [reply concepts/webhooks-lifecycle.md]: @Lauragra thanks for making the change!

File: concepts/changelog.md (hunk 1 above)
Comment (angelgolfer-ms) [reply concepts/changelog.md]: Ok, I see where our disconnect is - I searched the changelog, it only mentions in Dec. 2019, chatMessage started supporting notifications with resource data. There's no separate mention of lifecycle notifications for chatMessage.

**Review by angelgolfer-ms — APPROVED** [review]
Thanks for doc updates and clarification.

**Pull request merged**